packages/oui-criteria-container/src/criteria-container.component.js

```diff
@@ -0,0 +1,11 @@
+import controller from "./criteria-container.controller";
+
+export default {
+    template: "<ng-transclude></ng-transclude>",
+    transclude: true,
+    controller,
+    bindings: {
+        criteria: "=model",
```

Comment:
As discussed, I suggest to remove the two-way to respect the initial decision "Use two-way bindings with ng-model and one-way bindings for read-only components"